### ISSUE TITLE
[WHL] output sound is choppy if we play with paplay

### ISSUE THREAD

**Issue by bardliao** [thesofproject/linux#812]
kernel: f0ce2c52a7d37b22cb75ee7cf537eb067a34445b and cherry-pick  4b3555a886f990a66182d6f041f0cdda0f95aca1 + 75d88bd1124aac120980926d1a45048d738a3574
FW: https://github.com/thesofproject/sof/commit/ed6ac05f6b884b5aa5597bf739fc97527bdab909 and revert https://github.com/thesofproject/sof/commit/f5e78430ff3dbbe8bcceca5f0f4f1fa86bbd78b8

We can see a lot of log with same DMA and DAI position.
```
[  688.768510] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.768632] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.768642] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.768955] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.768999] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.769201] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.769231] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.769313] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.769331] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.798577] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
[  688.798605] sof-audio-pci 0000:00:1f.3: PCM: stream 0 dir 0 DMA position 12192 DAI position 56760
```
It seems we don't get SOF_IPC_STREAM_POSITION ipc in time.

How to reproduce:
1. boot up on a ubuntu device.
2. paplay test.wav

**Comment by bardliao** [comment]
This issue can be seen on both WHL HDA and I2S mode, but I can't reproduce it on Up2 (APL).

**Comment by mengdonglin** [comment]
Closed since PR #826 is merged and can fix this issue
